package.json

```diff
@@ -93,12 +93,12 @@
         "webpack-cli": "^3.3.10"
     },
     "peerDependencies": {
-        "react": "^15.0.1 || ^16.0.1",
-        "react-dom": "^15.0.1 || ^16.0.1"
+        "react": "^16.8",
```

Comment:
This is probably compatible with newer React too, but I'm not sure how to test that.